### PR TITLE
Display dynamic value

### PR DESCRIPTION
**Pull request by christophscheuing**
closes: #1806

### Visible/Frontend Changes
- added new component to show a dynamically calculated percentage from two values of the given entity

### CODE REVIEW

File: src/app/child-dev-project/children/health-checkup/health-checkup-component/health-checkup.component.ts [hunk 1]
```diff
@@ -32,6 +32,16 @@ export class HealthCheckupComponent implements OnInit {
...
+      {
+        id: "display-percentage",
+        label: "display-percentage",
+        view: "DisplayDynamicPercentage",
+        additional: {
+          actual: "weight",
+          total: "height",
+          decimalPlaces: 0,
+        },
+      },
```
Comment (sleidig) [inline src/app/child-dev-project/children/health-checkup/health-checkup-component/health-checkup.component.ts:44]: removing test code before merge
```suggestion
```

File: src/app/core/basic-datatypes/entity/display-entity/display-entity.stories.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ const Template: StoryFn<DisplayEntityComponent> = (
 });
 
 const testChild = new Child();
-testChild.name = "Test Name";
+testChild.name = "Test NameXXX";
```
Comment (sleidig) [inline src/app/core/basic-datatypes/entity/display-entity/display-entity.stories.ts:27]:
```suggestion
testChild.name = "Test Name";
```

**Review by sleidig — COMMENTED** [review]
looks good, thanks @christophscheuing !

I noticed this is not updating its value when you edit the related values (e.g. the "total" property of the entity), however. Seems a bit messy with our system, onChanges is not firing for this ...
I'll look into it further.

**Comment by TheSlimvReal** [comment]
The only way I was able to solve the issue with the change detection was by always re-calculating the result.

**Review by sleidig — APPROVED** [review]

**Pull request merged**